test/distributed-ranges/shp/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+if(${CMAKE_VERSION} VERSION_LESS "3.26")
+    message(WARNING "disabled distributed ranges tests, use cmake>=3.26")
+    return()
+else ()
+    message(STATUS "suitable for distributed ranges cmake version ${CMAKE_VERSION} found")
+endif()
+
+find_package(fmt QUIET)
+if(fmt_FOUND)
+    if(fmt_VERSION VERSION_GREATER_EQUAL 8.1 AND fmt_VERSION VERSION_LESS 10)
+        message(STATUS "suitable for distributed ranges libfmt version ${fmt_VERSION} found")
+    else()
+        message(WARNING "libfmt version ${fmt_VERSION} is not >= 8.1 and < 10, skipping distributed ranges tests")
+        return()
+
+    endif()
+else()
+    message(WARNING "libfmt not found, install libfmt-dev 8.x or 9.x to enable distributed ranges tests")
+    return()
+endif()
+
+find_package(GTest)
+if (NOT GTest_FOUND)
+  message(WARNING "disabled distributed ranges tests, install libgtest-dev or other package with GTest")
+  return()
+endif ()
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_executable(
+  shp-tests
+  shp-tests.cpp 
+  ../common/all.cpp 
+  ../common/copy.cpp 
+  ../common/counted.cpp
+  ../common/distributed_vector.cpp 
+  ../common/drop.cpp
+  ../common/enumerate.cpp
+  ../common/fill.cpp 
+  ../common/for_each.cpp
+  ../common/iota.cpp
+  # ../common/iota_view.cpp # compile fail, even in distributed-ranges project
```

Comment:
Perhaps this would be better to be controlled / skipped within the test itself maybe with some warning / description rather than here in the CMake file with a comment?

Reply:
applied